#feedback_int.msg

int8 mode
int16 encoderR
int16 encoderL
int16 pwm_to_motorR
int16 pwm_to_motorL
uint8 looptime
int16 analogData0
int16 analogData1


